src/github/branch.test.ts

```diff
@@ -1,164 +1,78 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
-import { createWebhookApp } from "test/utils/probot";
```

Comment:
Purely move the file to a proper location, and use the same name for unit testing.